edb/server/protocol/auth_ext/_static/webauthn-authenticate.js

```diff
@@ -150,10 +150,12 @@ async function authenticateAssertion(props) {
   // Assertion includes raw bytes, so need to be encoded as base64url
   // for transmission
   const encodedAssertion = {
-    ...props.assertion,
+    type: props.assertion.type,
+    id: props.assertion.id,
+    authenticatorAttachments: props.assertion.authenticatorAttachment,
```

Comment:
Should this be `authenticatorAttachment` (no 's')?

Reply:
Good eye. It's optional so I guess it worked in my testing just accidentally 🙈 . Getting pretty close to creating a mock authenticator so we can actually test these end-to-end...